lua/entities/gmod_door_exterior/init.lua

```diff
@@ -87,11 +87,17 @@ function ENT:Think()
             self.occupants[k]=nil
         end
     end
-    
+
     self:CallHook("Think",CurTime()-self.lastthink)
     self.lastthink=CurTime()
     if self:CallHook("ShouldThinkFast") then
         self:NextThink(CurTime())
         return true
     end
-end
\ No newline at end of file
+
+    if not self._init then return end
+    if CurTime() >= (self.nextslowthink or 0) then
```

Comment:
Can you switch this to use the same as the `self.lastthink` so define it in the `ENT:Initialize` so we don't need the fallback of 0. Every little bit of performance helps in a Think hook!